source/server/core_server.cpp

```diff
@@ -266,7 +268,7 @@ int main(int argc, char** argv)
 #if defined(_WIN32)
   nidevice_grpc::set_console_ctrl_handler(&StopServer);
 #else
-  if (ni::data_monikers::is_sideband_streaming_enabled(config.feature_toggles)) {
+  if (ni::data_monikers::is_moniker_streaming_enabled(config.feature_toggles)) {
     SysFsWrite("/dev/cgroup/cpuset/system_set/cpus", "0-5");
```

Comment:
Should we remove this hardcoded values  assigning cpus?

Reply:
Yes. I think they got left out when Raghav removed configuration support for core server cpu affinity. I will create separate PR for better tracking and merge that as well in some time.